spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/vision/CloudVisionProperties.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Copyright 2018 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.autoconfigure.vision;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.NestedConfigurationProperty;
+import org.springframework.cloud.gcp.core.Credentials;
+import org.springframework.cloud.gcp.core.CredentialsSupplier;
+import org.springframework.cloud.gcp.core.GcpScope;
+
+@ConfigurationProperties("spring.cloud.gcp.vision")
+public class CloudVisionProperties implements CredentialsSupplier {
+
+	/** Overrides the GCP OAuth2 credentials specified in the Core module. */
```

Comment:
The Javadoc must be really Javadocs, so, from new line:
```suggestion
	/** 
	* Overrides the GCP OAuth2 credentials specified in the Core module. 
	*/
```

Reply:
Done, I downgraded to inline comment since I felt it was not significant enough to be javadoc.